### PR TITLE
TPDMDEV-195 - v1.0 Education Organization Updates

### PR DESCRIPTION
**Pull request by stevenarnold-ed-fi**

### CODE REVIEW

File: Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql [hunk 1]
```diff
@@ -1,8 +1,3 @@
--- SPDX-License-Identifier: Apache-2.0
```
Comment (blmeyers) [inline Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql:1]: Losing the Ed-Fi license on these scripts.  By default MetaEd will remove these for extensions, might want to raise a ticket to handle this, it should probably generate copyright headers even on extension files if you run it in Alliance mode.  For now you'll have to just make sure they don't get removed.

**Review by blmeyers — CHANGES_REQUESTED** [review]
Fails during initdev.  Looks like University has been removed, but the scripts at https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions/tree/main/Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods for setting up security views is still trying to use it.

Will also need to update https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions/blob/main/Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1010-TPDM-ResourceClaims.sql based on new/removed resources.

File: Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql (hunk 1 above)
Comment (vimayya) [reply Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql:1]: @stevenarnold-ed-fi there is already a ticket for it https://tracker.ed-fi.org/browse/METAED-1152, could you add a comment about including extensions when run in Alliance mode.

Comment (stevenarnold-ed-fi) [reply Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Structure/Ods/0010-EXTENSION-TPDM-Schemas.sql:1]: @blmeyers  @vimayya Updated the security views and removed files related only to the ed-orgs that were removed. Also updated the claims to remove ed-orgs. Ran the EdFi.Ods.Extensions.TPDM TeamCity job on the branch and ran an initdev -UsePlugins to load the updated extension. Built and runs, all looks good.

**Review by blmeyers — APPROVED** [review]

**Comment by blmeyers** [comment]
Changes look good now, @vimayya We'll need a ticket at some point to clean up the work done as part of https://tracker.ed-fi.org/browse/ODS-4018, since the ed org sub types that work was done for were removed as part of this PR.

**Pull request merged**